Makefile

```diff
@@ -41,8 +41,10 @@ publish:
 	git push origin master
 	git tag $(NEW_VERSION_TAG)
 	git push origin $(NEW_VERSION_TAG)
+
+publish:
 	brew update && brew bump-formula-pr --tag=$(NEW_VERSION_TAG) --revision=$(shell git rev-parse HEAD) needle
```

Comment:
`NEW_VERSION_TAG` is empty in this path right?

Reply:
Yep fixed